sdk/trace/span_processor_example_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"time"
+
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	"go.opentelemetry.io/otel/sdk/export/trace/tracetest"
+)
+
+// LowPassFilter is a SpanProcessor that drops short lived spans.
```

Comment:
Shouldn't these names be reversed?  A low-pass filter should allow signal below the threshold to pass through. To carry the analogy through here I'd expect a low-pass filter to allow spans with a duration below the threshold to pass through.

Reply:
My signal processing roots are showing through here. A low-pass filter will allow low *frequency* signals below a cutoff frequency to pass through unattenuated. The duration of the span being analogous to the period of a signal it is inversely related to the frequency, meaning that low frequency spans would be have long durations and high frequency spans short durations.

I can try to find better names if you think this will overall be more confusing than helpful.

Reply:
The explanation you provide makes sense when frequency is the measured value.  I do think it will likely be non-intuitive, though, if the measured value is duration and the relationship is inverted with respect to a straight reading of the name.  I see a "low-pass" filter with configuration for a duration and expect it will allow signals below that duration to pass through.

Perhaps the suggestion from Wikipedia can provide a guide: 

>In optics, high-pass and low-pass may have different meanings, depending on whether referring to frequency or wavelength of light, since these variables are inversely related. High-pass frequency filters would act as low-pass wavelength filters, and vice versa. For this reason it is a good practice to refer to wavelength filters as "Short-pass" and "Long-pass" to avoid confusion, which would correspond to "high-pass" and "low-pass" frequencies

Reply:
Given the goal here is to show how to use the `SpanProcessor`s I'm going to update this to have each processor act in a different way. One will be to filter based on duration and the other will filter based on instrumentation. I think that will provide a more informative example.